src/integrationtest/python/DocTraversals_tests.py

```diff
@@ -0,0 +1,47 @@
+# Name: Debasish Kanhar
+
+import unittest
+from subprocess import Popen, PIPE
+from JanuaGraphContainer import JanusGraphContainer
+from janusgraph_python.driver.ClientBuilder import JanusGraphClient
+from gremlin_python.structure.graph import Graph
+
+
+class TestDocTraversals(unittest.TestCase):
+    def setUp(self):
+        self.container = JanusGraphContainer()
+
+        docker_ip = Popen(["docker-machine", "ip"], stdout=PIPE).communicate()[0]
+        docker_ip = docker_ip.strip().decode("utf-8")
+
+        self.client = JanusGraphClient()
+        self.client = self.client.connect(host=str(docker_ip), port="8182",
```

Comment:
Are you able to combine these two variable assignments of `self.client` into one line?